cpp/Platform.Timestamps/Timestamp.h

```diff
@@ -1,6 +1,6 @@
 ﻿#pragma once
 
-#include <functional>
+#include <typeindex>
```

Comment:
Why?

Reply:
Because it declares the same std::hash function, but with less other stuff included.